src/x509/clu_x509_sign.c

```diff
@@ -208,6 +212,736 @@ void wolfCLU_CertSignSetCA(WOLFCLU_CERT_SIGN* csign, WOLFSSL_X509* ca,
     }
 }
 
+/* ref: https://github.com/wolfssl/wolfssl-examples/X9.146/gen_ecdsa_mldsa_dual_keysig_cert.c */
+void wolfCLU_GenChimeraCertSign(WOLFSSL_BIO *bioCaKey, WOLFSSL_BIO *bioAltCaKey,
+        WOLFSSL_BIO *bioAltSubjPubKey, WOLFSSL_BIO *bioSubjKey,
+        WOLFSSL_X509 *caCert, const char *subject,
+        const char *outFileName, int outForm)
+{
+#if defined(WOLFSSL_DUAL_ALG_CERTS) && defined(HAVE_DILITHIUM)
+    int ret = WOLFCLU_SUCCESS;
+    int isCA = 0;
+
+    WC_RNG   rng;
+    ecc_key  caKey;
+    ecc_key  serverKey;
+    MlDsaKey altCaKey;
+    word32 idx = 0;
+    byte level = 0;
+
+    Cert newCert;
+    DecodedCert preTBS;
+
+    int initRNG       = 0;
+    int initCaKey     = 0;
+    int initServerKey = 0;
+    int initAltCaKey  = 0;
+    int initPreTBS    = 0;
+
+    XFILE caKeyFp       = NULL;
+    XFILE altCaKeyFp    = NULL;
+    XFILE altCaPubKeyFp = NULL;
+    XFILE serverKeyFp   = NULL;
+    WOLFSSL_BIO *out    = NULL;
+
+    char *token   = NULL;
+    char *key     = NULL;
+    char *value   = NULL;
+    char *saveptr = NULL;
+    char *subj    = NULL;
+    int  subjSz   = 0;
+
+    const int LARGE_TEMP_SZ = 11264;
+    byte caKeyBuf[LARGE_TEMP_SZ];
+    int  caKeySz   = LARGE_TEMP_SZ;
+    byte altCaKeyBuf[LARGE_TEMP_SZ];
+    int  altCaKeySz = LARGE_TEMP_SZ;
+    byte sapkiBuf[LARGE_TEMP_SZ];
+    int  sapkiSz = LARGE_TEMP_SZ;
+    byte altSigAlgBuf[LARGE_TEMP_SZ];
+    int  altSigAlgSz = LARGE_TEMP_SZ;
+    byte scratchBuf[LARGE_TEMP_SZ];
+    int  scratchSz = LARGE_TEMP_SZ;
+    byte preTbsBuf[LARGE_TEMP_SZ];
+    int  preTbsSz = LARGE_TEMP_SZ;
+    byte altSigValBuf[LARGE_TEMP_SZ];
+    int  altSigValSz = LARGE_TEMP_SZ;
+    byte derBuf[LARGE_TEMP_SZ];
+    int  derSz = LARGE_TEMP_SZ;
+    byte outBuf[LARGE_TEMP_SZ];
+    int  outSz = LARGE_TEMP_SZ;
```

Comment:
This is a massive stack. You mention in the description tackling small stack builds later which is okay. Just making note here for later optimization.